src/System.Runtime/tests/System/StringsMatchingNonRandomizedHashCode.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Tests
+{
+    internal static class StringsMatchingNonRandomizedHashCode
+    {
+        internal static readonly string[] Data = new string[102]
```

Comment:
`// A list of strings that all have the same hashcode (-1038491930) using the legacy non randomized string hash code algorithm`